src/v/config/configuration.cc

```diff
@@ -593,6 +593,12 @@ configuration::configuration()
       {.example = "32768", .visibility = visibility::tunable},
       16_KiB,
       storage::internal::chunk_cache::validate_chunk_size)
+  , segment_fallocation_step(
+      *this,
+      "segment_fallocation_step",
+      "Size for segments fallocation",
+      {.example = "32768", .visibility = visibility::tunable},
```

Comment:
This should be relatively easy to make live-adjustable:
- Consume it via the .bind() method and pass around a `binding` rather than an integer (https://vectorizedio.atlassian.net/wiki/spaces/CORE/pages/78676076/Configuration+Properties)
- Once that's done, set ".needs_restart = needs_restart::no" here.